ivy/functional/ivy/layers.py

```diff
@@ -697,16 +696,36 @@ def conv2d(
               [[2.],[4.],[6.]]
               ]])
 
-    With :code:`ivy.NativeArray` input:
-
-    >>> x = ivy.native_array(ivy.random_normal(mean=0, std=1, shape=[1, 32, 32, 3]))
-    >>> filters = ivy.native_array(ivy.random_normal(mean=0, std=1, \
-                                   shape=[3, 5, 3, 5])) #HWIO
-    >>> result = ivy.conv2d(x, filters, [2, 1], 'VALID') \
-        #non-square filter with unequal stride and valid padding
-    >>> print(result.shape)
-    (1, 15, 28, 5)
+    With one :code:`ivy.Container` input:
+    
+    >>> x = ivy.Container(a=ivy.array([[[[1.], [2.0],[3.]], \
+                                        [[1.], [2.0],[3.]], \
+                                        [[1.], [2.0],[3.]]]]))
+    >>> filters = ivy.eye(3, 3).unsqueeze(-1).unsqueeze(-1).float()
+    >>> result = ivy.conv2d(x, filters, (2,), 'SAME', data_format='NHWC', \
+        dilations= (1,))
+    >>> print(result)
+    {
+        a:ivy.array([[[[3.], [3.]], [[1.], [5.]]]])
+    }
 
+    With multiple :code:`ivy.Container` inputs:
+    >>> x = ivy.Container(a = (3, 3).view(1, 3, 3, 1),  \
```

Comment:
`view` isn't a function in Ivy's API, I'd suggest you to use functions present in ivy and also test your examples to see if the results are as expected